server/routes/pages.js

```diff
@@ -57,6 +59,12 @@ async function handlePage(req, res) {
 
   if (page === 'categories' || page === 'index') {
     const tree = await getTree()
+    if (!tree.children) {
```

Comment:
👍  This signal is great. It also seems like we could easily make this nonfatal (log, but have the library site be empty) if we wanted.